Core/include/Acts/Surfaces/detail/VerticesHelper.hpp

```diff
@@ -81,11 +45,11 @@ static std::vector<double> phiSegments(double phiMin = -M_PI,
 /// @param offset The out of plane offset position of the bow
 /// @param transform The transform applied (optional)
 template <typename vertex_t, typename transform_t>
-void createSegment(std::vector<vertex_t>& vertices,
-                   std::pair<double, double> rxy, double phi1, double phi2,
-                   unsigned int lseg, int addon = 0,
-                   const vertex_t& offset = vertex_t::Zero(),
-                   const transform_t& transform = transform_t::Identity()) {
+inline void createSegment(
+    std::vector<vertex_t>& vertices, std::pair<double, double> rxy, double phi1,
+    double phi2, unsigned int lseg, int addon = 0,
+    const vertex_t& offset = vertex_t::Zero(),
+    const transform_t& transform = transform_t::Identity()) {
```

Comment:
These I don't get. I still think template functions don't need `inline`, and I feel like this is just noisy.

Reply:
Fixed.